lib/commands/install.js

```diff
@@ -140,6 +141,21 @@ class Install extends ArboristWorkspaceCmd {
       throw this.usageError()
     }
 
+    if (!isGlobalInstall) {
+      const pkgJson = new PackageJson()
+      await pkgJson.load(this.npm.prefix)
+      const jsonContent = pkgJson.content
+      const isJsonEmptyObject = Object.keys(jsonContent).length === 0
+        && jsonContent.constructor === Object
+
+      const emptyJson = new Error('EEMPTYPKGJSON: package.json is empty object')
+      emptyJson.code = 'EEMPTYPKGJSON'
+      emptyJson.path = pkgJson.filename
+      if (isJsonEmptyObject) {
+        throw emptyJson
+      }
```

Comment:
Hi, @wraithgar, **that code** do you mean this snippet? But it's written by me, not left over before I wrote the commit? I still don't understand, Can you point to one place of the code for example?

Reply:
I was trying to infer why the test for an empty object was there and assumed it was due to `rpj`.  The long and the short of it is we don't need that check, either `package-json` returns or it doesn't, if it returns at all we read a valid package.json from somewhere in the tree.